README.md

```diff
@@ -87,3 +87,17 @@ $ yarn lint-fix
 ```
 
 which will directly fix the files that didn't pass.
+
+## Using a different Pollbot server
```

Comment:
nit: This is actually only applicable for developers of delivery-dashboard. 

There should be another section here in the README for **users** of delivery-dashboard who wants to test against a different Pollbot other than pollbot.services.mozilla.com.